src/ui/views/Table/components/DataGrid/GridRow.svelte

```diff
@@ -54,34 +53,9 @@
       }
     };
   }
-
-  function handleHoverLink(event: MouseEvent) {
-    if (!event.ctrlKey && !event.metaKey) {
-      return;
-    }
```

Comment:
With regard to this condition, the `default mod` option should be true to get better backwards-compatibility.

Reply:
Sounds good